web/channels/user_socket.ex

```diff
@@ -5,7 +5,8 @@ defmodule ColaboraApi.UserSocket do
   # channel "rooms:*", ColaboraApi.RoomChannel
 
   ## Transports
-  transport :websocket, Phoenix.Transports.WebSocket
+  transport :websocket, Phoenix.Transports.WebSocket,
+    timeout: 45_000
   # transport :longpoll, Phoenix.Transports.LongPoll
```

Comment:
Apparently heroku times out websockets connections after 55 seconds. This way you ensure that your app is handling the timeouts instead of heroku.

At least that's what i understood from the phoenix deployment guides...